sky/provision/kubernetes/utils.py

```diff
@@ -1851,15 +1851,19 @@ def get_kubernetes_node_info(
                         if container.resources.requests:
                             allocated_qty += int(
                                 container.resources.requests.get(
-                                    'nvidia.com/gpu', 0))
+                                    common_utils.get_gpu_resource_name(), 0))
             accelerators_available = accelerator_count - allocated_qty
 
         node_info_dict[node.metadata.name] = KubernetesNodeInfo(
             name=node.metadata.name,
             gpu_type=accelerator_name,
-            total={'nvidia.com/gpu': int(accelerator_count)},
-            free={'nvidia.com/gpu': int(accelerators_available)})
-
+            total={
+                common_utils.get_gpu_resource_name(): int(accelerator_count)
+            },
+            free={
+                common_utils.get_gpu_resource_name():
+                    int(accelerators_available)
+            })
```

Comment:
Cleaner to invoke once:
```suggestion
            gpu_resource = common_utils.get_gpu_resource_name()
            total={gpu_resource: int(accelerator_count)},
            free={gpu_resource: int(accelerators_available)
            })
```

Reply:
done